class-12-REST-and-APIs/pair-assignment/starter-code/scripts/aboutController.js

```diff
@@ -7,7 +7,10 @@
     // REVIEW: Look at this method chaning. What is being accomplished here?
     $('#about').show().siblings().hide();
 
-    // TODO: Call a function to 'request' our repo data.
+    // TODONE: Call a function to 'request' our repo data.
```

Comment:
no need to wrap repoView.index in an anonymous function here. just repos.requestRepos(repoView.index) should do the trick